### PR TITLE
Some cleanup for the bootspec class

### PR DESCRIPTION
**Pull request by ejoerns**
* remove True option from getVar() calls
* rename 'dtb' to 'dtbs' to reflect it holds multiple device trees
* do not strip .dtb ending from dtbs list and re-append it later on
  Instead, strip it only when creating the .conf filename
* Use variable for conf filename to reduce code duplication

### CODE REVIEW

File: classes/bootspec.bbclass [hunk 1]
```diff
@@ -36,15 +36,17 @@ BOOTSPEC_EXTRALINE ?= ""
 BOOTSPEC_EXTRALINE[doc] = "Allows to add extra content to bootspec entries, lines must be terminated with a newline"
 
 python create_bootspec() {
-    dtb = (d.getVar('KERNEL_DEVICETREE') or "default").replace('.dtb', '').split()
```
Comment (medude) [inline classes/bootspec.bbclass]: Shouldn't we split this commit in two?

**Review by medude — CHANGES_REQUESTED** [review]

File: classes/bootspec.bbclass (hunk 1 above)
Comment (ejoerns) [reply classes/bootspec.bbclass]: No strong opinion on that. A separate one for the 'dtbs'-Rename? Or for the conf variable?

Comment (medude) [reply classes/bootspec.bbclass]: That would be cool :)

Comment (ejoerns) [reply classes/bootspec.bbclass]: I like people answering with 'yes' on an 'or' question! ;)

Comment (medude) [reply classes/bootspec.bbclass]: Damn. I mean: one patch for the conf in front of the renaming.

Comment (ejoerns) [reply classes/bootspec.bbclass]: Ok, did that.

**Review by medude — APPROVED** [review]

**Comment by ejoerns** [comment]
Cherry-picked to `dunfell` branch.

**Pull request merged**